test/jdk/build/AbsPathsInImage.java

```diff
@@ -164,7 +164,7 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
                 String fileName = file.toString();
                 if (Files.isSymbolicLink(file)) {
                     return super.visitFile(file, attrs);
-                } else if (fileName.endsWith(".debuginfo") || fileName.endsWith(".pdb")) {
+                } else if (fileName.endsWith(".pdb")) {
```

Comment:
The .debuginfo suffix has at least historically been used on more OSes than just Linux. I think we should only include .debuginfo files in this test if the OS is also Linux.

Reply:
@erikj79 As far as I can see only AIX is the exception of not being Linux and using .debuginfo suffix, are there others?
I am currently testing AIX, so I will see if it passes, as you say, I would have my doubts it will!

Reply:
Solaris used that extension as well. It's no longer supported in mainline, but if you backport this, which I assume you will want to, then someone may hit it.

Reply:
ah right yes Solaris
I'll add a suitable  System.getProperty("os.name").contains("Linux")